sampler.go

```diff
@@ -36,6 +36,7 @@ const (
 	defaultSamplingServerURL       = "http://localhost:5778/sampling"
 	defaultSamplingRefreshInterval = time.Minute
 	defaultMaxOperations           = 2000
+	defaultUpperBound              = 2 // Units in seconds, ie 2 per second
```

Comment:
maybe more explicit `defaultMaxSamplesPerSecond`, and change upperBound/lowerBound fields to minSamplesPerSecond/maxSamplesPerSecond